mystyle.css

```diff
@@ -32,26 +32,24 @@ nav {
 }
 
 /*List Members Bar style*/
-#listMembersBar {
+.list-members-bar,
+.add-items-bar {
     margin-bottom: 20px;
     margin-left: 20px;
 }
 
 /*Add Item Bar style*/
-    /*Text input*/
-    #text-input {
-        float: left;
```

Comment:
This `float: left` isn't necessary. Floats are kinda weird, so it's best not to have extraneous ones sitting around

The HTML you had was something like
```
<div class="container-fluid">
    <div class="col-xs-9"></div>
    <div class="col-xs-3"></div>
</div>
```
and all those classes are taken care of Bootstrap to lay it out the way you want without this extra `float: left`